src/parser/shaman/enhancement/modules/core/Stormbringer.js

```diff
@@ -0,0 +1,32 @@
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SPELLS from 'common/SPELLS/shaman';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+
+class Stormbringer extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+  };
+
+  constructor(...args){
+    super(...args);
+    this.addEventListener(Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.STORMBRINGER), this.onReset());
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.STORMSTRIKE), this.onNoCooldown());
+  }
+
+  onReset(event) {
+    this.spellUsable.endCooldown(SPELLS.STORMSTRIKE.id);
+  }
+
+  onNoCooldown() {
+    if(!this.selectedCombatant.hasBuff(SPELLS.STORMBRINGER.id)){
```

Comment:
You could simply this if statement by inverting it.
Also, I am not sure how strict the formatting standards are here, but this statement is missing some whitespaces :)

```
if (this.selectedCombatant.hasBuff(SPELLS.STORMBRINGER.id)) {
    this.spellUsable.endCooldown(SPELLS.STORMSTRIKE.id);
}

Reply:
Usually we prefer early returns over nested statements.
I mean here it is pretty trivial anyway but if the code starts getting more complex, you will run into issues with having it nested that way.

![grafik](https://user-images.githubusercontent.com/2129161/53257624-ee405d00-36ca-11e9-8e4a-50e14da894fa.png)
(Holy paladin is maintained by the boss so I usually refer to his stuff regarding code style.
Let's take that as reference. If you would nest everything here it would start getting kinda messy.

Regarding the whitespaces, what I have seen most people do is use whitespaces for comparing, objects but not for functions and such. Refer to the above screenshot for the same thing with the functions for the first if clause. 

And here you can see the whitespaces for the curly braces I am refering to.
![grafik](https://user-images.githubusercontent.com/2129161/53257963-b1c13100-36cb-11e9-8b38-89e5bd94398d.png)